src/links/_base.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file Link
+ * @module Link
+ * @overview Link and top of page featured link modules
```

Comment:
Not sure what page featured link modules means. Perhaps we can clarify this a bit so there is less ambiguity for devs.